Xamarin.Forms.ControlGallery.WPF/Xamarin.Forms.ControlGallery.WPF.csproj

```diff
@@ -56,6 +56,9 @@
     <PackageReference Include="Microsoft.Maps.MapControl.WPF" Version="1.0.0.3" />
     <PackageReference Include="OpenTK" Version="3.0.1" />
     <PackageReference Include="OpenTK.GLControl" Version="3.0.1" />
+    <PackageReference Include="Xamarin.Essentials">
```

Comment:
WPF is not supported. Do we need it?

Reply:
There is currently no WPF support this is not needed, I can pull this out.

Reply:
This was resolved when I rebased the branch on 4.6.0